src/services/slack.ts

```diff
@@ -191,13 +191,23 @@ export class SlackConnector implements ISlackConnector {
       };
       reposToShow.push(opt);
     });
+
     // THis is the limitation enforced by slack as no more 100 items are allowd in the dropdown
+    //Sort the list so that any inactive versions are at the end and will be truncated if any items must be truncated
     //'[ERROR] no more than 100 items allowed [json-pointer:/view/blocks/0/element/options]'
 
     if (reposToShow.length > 100) {
-      reposToShow = reposToShow.splice(0, 100);
+      reposToShow = reposToShow.sort().reverse().splice(0, 100);
```

Comment:
Normally using a sort method without a compare function makes me sweat but simply because Github enforces rules on owner names and repo names in respect to symbols, this is safe to me. I'm 99.99% sure.

Reply:
Hm ok, do you think I should add a sort function of some ~sort?

Reply:
This is going to be the very rare time when I say you don't have to 😆